README.md

```diff
@@ -13,10 +13,13 @@ Oculus Legacy  (Linux, Mac, Windows)
 Oculus (Windows)
 * Oculus 0.7 runtime
 
-Note
-====
-* Windows 64 builds are not working at the moment, use 32 bits instead
-* Extended Mode is the only supported mode
+HTC Vive (Windows)
+* OpenVR 0.9.20 runtime
+
+Current Known Issues
+====================
+HTC Vive: TODO: Fix Blender crash on OpenVR when stopping. (Pausing works).
```

Comment:
I did fix this crash on the commit https://github.com/cedeon/hmd_sdk_bridge/commit/e70a76e2929385eb5a55d8bf6941d83a576b6f49 . It was simply a missing return statement.  I need to remove this from the readme.